src/http/http_server.cpp

```diff
@@ -64,13 +79,7 @@ http_server::http_server() : serverlet<http_server>("http_server")
     tools::register_message_header_parser<http_message_parser>(NET_HDR_HTTP, {"GET ", "POST"});
 
     // add builtin services
-    register_http_service(new root_http_service());
-
-#ifdef DSN_ENABLE_GPERF
-    register_http_service(new pprof_http_service());
```

Comment:
I don't find out where to register pprof http serivce, do we need to register it?

Reply:
```
/*extern*/ void register_builtin_http_calls()
{
#ifdef DSN_ENABLE_GPERF
    static pprof_http_service pprof_svc;
#endif
```

No. The constructor of pprof_http_service actually does the registration.